app/controllers/reports/regions_controller.rb

```diff
@@ -14,15 +14,16 @@ def index
     if current_admin.feature_enabled?(:region_reports)
       accessible_facility_regions = authorize { current_admin.accessible_facility_regions(:view_reports) }
       cache_key = "#{current_admin.cache_key}/regions/index"
-      cache_version = accessible_facility_regions.cache_key
+      cache_version = "#{accessible_facility_regions.cache_key} / v2"
```

Comment:
This change ensure we bust the index view cache when we deploy, right?

Shall we strip the spaces out of the cache version for idiomatic-ness?

Reply:
Yup, and yup. :grin